examples/development/platform_integration/lib/generated_pigeon.dart

```diff
@@ -1,16 +1,117 @@
-// #docregion Search
```

Comment:
Is this going to break a snippet somewhere?

Reply:
I updated the contents in this file and moved them to `pigeon_source.dart` where the updated docregion is.